autoload/lsp.vim

```diff
@@ -516,16 +520,14 @@ function! s:ensure_init(buf, server_name, cb) abort
     let l:server_info = l:server['server_info']
     if has_key(l:server_info, 'root_uri')
         let l:root_uri = l:server_info['root_uri'](l:server_info)
-    else
-        let l:root_uri = lsp#utils#get_default_root_uri()
     endif
 
     if empty(l:root_uri)
```

Comment:
This could cause error.
`l:root_uri` should initialize in above of this line, I think.

Reply:
Right